cli/command/service/scale.go

```diff
@@ -72,17 +78,14 @@ func runServiceScale(dockerCli *command.DockerCli, serviceID string, scale strin
 	if serviceMode.Replicated == nil {
 		return fmt.Errorf("scale can only be used with replicated mode")
 	}
-	uintScale, err := strconv.ParseUint(scale, 10, 64)
-	if err != nil {
-		return fmt.Errorf("invalid replicas value %s: %s", scale, err.Error())
-	}
+
```

Comment:
Thinking if simply moving this check "up" inside the function, instead of moving it outside would be cleaner (open for input from other maintainers here)